RELEASING.md

```diff
@@ -67,6 +67,11 @@ We use `go version` to generate the SBOM for Atlas CLI binaries. You can generat
 go version -m <path_to_atlasCLI_binary>
 ```
 
+## MongoCLI Binary
```

Comment:
I think this is already documented in the mongocli branch https://github.com/mongodb/mongodb-atlas-cli/blob/mongocli-master/RELEASING.md

Reply:
ah thanks!